internal/pkg/service/common/etcdop/etcdop.go

```diff
@@ -1,10 +1,23 @@
-// Package etcdop provides a small framework on top of etcd low-level operations.
+// Package etcdop provides a framework on top of etcd low-level operations.
 //
-// See Key and Prefix types. Examples can be found in the tests.
+// At first, create a custom prefix using NewPrefix/NewTypedPrefix functions.
+// Examples can be found in the tests. See also Key, KeyT[T], Prefix and PrefixT[T] types.
 //
 // Goals:
 // - Reduce the risk of an error when defining an operation.
 // - Distinguish between operations over one key (Key type) and several keys (Prefix type).
+// - Provides Serialization composed of encode, decode and validate operations.
```

Comment:
Pomocou generickych typov su hodnoty automaticky validovane a kodovane do/z JSON.

Reply:
Doplnil som trocha tento popis.

Reply:
(najlepsie je si to otvorit v IDE, snad je z kodu jasne, ako to funguje)